@ndlib/gatsby-theme-marble/src/components/Common/form.js

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react';
+import InputText from '../Common/inputText'
+import InputTextRadio from '../Common/inputRadio'
+import Recaptcha from 'react-google-invisible-recaptcha'
+
+import style from './style.module.css'
+import InputBox from './inputBox';
+
+class Form extends Component {
```

Comment:
This component is littered with linting violations that your IDE should really alert you to, which makes it much harder to ignore. It can also auto-fix certain errors on saving if setup correctly. All you really need is an eslint extension that should read the `.eslintrc` from the project root and display alerts in the gutter accordingly.

If you need help setting this up, I can probably assist, although I'm not as familiar with VS Code.